doc-content/enterprise-only/websphere/was-configure-embedded-engine-proc.adoc

```diff
@@ -0,0 +1,284 @@
+[id='was-configure-embedded-engine-proc']
+ifdef::PAM[]
+= Configuring an embedded {PROCESS_ENGINE} and {DECISION_ENGINE} in {WEBSPHERE}
+endif::[]
+
+ifdef::DM[]
+= Configuring an embedded {DECISION_ENGINE} in {WEBSPHERE}
+endif::[]
+
+ifdef::PAM[]
+An embedded engine is a light-weight work flow engine and rule engine that allows you to execute your decisions and business processes. An embedded engine can be part of your {PRODUCT_PAM} application or they can be deployed as a service, for example in the cloud. This section describes how to setup an embedded {PROCESS_ENGINE} or {DECISION_ENGINE} in {WEBSPHERE}.
```

Comment:
workflow and rule engine
Do not use allow to mean ability - I would use 'enables'
or it can be deployed

This procedure (not section)

Then you just in to the Maven stuff. I want you to have another think about this section. For example
:
To use the correct Maven dependencies in your Red Hat Process Automation Manager project,
You haven't talked about projects. How do projects relate to embedded engines, or applications? I don't want you to write an essay, in fact, the opposite. Just read through these paragraphs and ask yourself how you can make them flow better. Right now, I can see where you have copied and pasted.